### PR TITLE
feat: add recent PRs to PR Lookup

### PR DESCRIPTION
**Pull request by codebytere**
Improve UI of PR Lookup by adding recent PRs in the style of [ChromiumDash](https://chromiumdash.appspot.com/commits?platform=Android). Also improve some of the styling.

Screenshot 2024-12-10 at 3 47 25 PM

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -43,6 +45,21 @@ const getReleasesOrUpdate = pMemoize(
...
+      electron: electronJSON,
+      nightly: nightlyJSON,
+    };
+  },
+  {
+    cache: new ExpiryMap(60 * 1000),
+    cacheKey: () => 'download_stats',
+  },
+);
+
```
Comment (dsanders11) [inline src/data.js:62]: This looks unused?

File: src/data.js [hunk 2]
```diff
@@ -88,15 +105,40 @@ const getGitHubRelease = pMemoize(
...
+      const { data } = await (
+        await getOctokit()
+      ).pulls.list({
+        ...REPO_DATA,
+        state: 'closed',
+        base: 'main',
+      });
+      return data
+        .filter((pr) => {
+          const isBot = pr.user.login.endsWith('[bot]');
```
Comment (dsanders11) [inline src/data.js:120]:
```suggestion
          const isBot = pr.user.type === 'Bot';
```

**Review by dsanders11 — COMMENTED** [review]
Nice! Do you think it would be reasonable to tweak it so that clicking on a row takes you to the PR lookup for that PR? UX wise it's a bit unintuitive that I can't click the PR and have to manually type the number in to look it up. Maybe we could keep the link to the GitHub PR but put it as a GitHub logo icon next to the PR number and have that be the link?

File: src/data.js (hunk 1 above)
Comment (codebytere) [reply src/data.js:62]: @dsanders11 correct, leftover from another PR i was working on 😅

**Review by jkleinsc — APPROVED** [review]

**Pull request merged**